### PR TITLE
CIV-11161 - Docmosis: Manual Determination -"Claimant's Response"

### PR DESCRIPTION
**Pull request by kalachandrasekar1**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-11161

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by ElifDenizEsea — DISMISSED** [review]

**Review by ElifDenizEsea — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/manualdetermination/ClaimantLipManualDeterminationFormGenerator.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+@Getter
+@RequiredArgsConstructor
+public class ClaimantLipManualDeterminationFormGenerator implements TemplateDataGenerator<ClaimantLipManualDeterminationForm> {
+
+    private final DocumentManagementService documentManagementService;
+    private final DocumentGeneratorService documentGeneratorService;
+
+    public CaseDocument generate(CaseData caseData, String authorisation) {
+        ClaimantLipManualDeterminationForm templateData = getTemplateData(caseData);
+
```
Comment (jarekPierchala) [inline src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/manualdetermination/ClaimantLipManualDeterminationFormGenerator.java:35]: no need for these empty lines. Context of the code for document and templates is the same.

Comment (kalachandrasekar1) [reply src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/manualdetermination/ClaimantLipManualDeterminationFormGenerator.java]: fixed

File: src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/manualdetermination/ClaimantLipManualDeterminationFormGenerator.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        DocmosisTemplates template = getDocmosisTemplate();
+
+        DocmosisDocument docmosisDocument = documentGeneratorService.generateDocmosisDocument(templateData, template);
+
+        return documentManagementService.uploadDocument(
+                authorisation,
+                new PDF(getFileName(template, caseData), docmosisDocument.getBytes(), DocumentType.LIP_MANUAL_DETERMINATION));
```
[the remark this answers is not in the document]
Comment (kalachandrasekar1) [reply src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/manualdetermination/ClaimantLipManualDeterminationFormGenerator.java]: fixed

File: src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/manualdetermination/ClaimantLipManualDeterminationFormGenerator.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+public class ClaimantLipManualDeterminationFormGenerator implements TemplateDataGenerator<ClaimantLipManualDeterminationForm> {
+
+    private final DocumentManagementService documentManagementService;
+    private final DocumentGeneratorService documentGeneratorService;
+
+    public CaseDocument generate(CaseData caseData, String authorisation) {
+        ClaimantLipManualDeterminationForm templateData = getTemplateData(caseData);
+
+        DocmosisTemplates template = getDocmosisTemplate();
+
```
[the remark this answers is not in the document]
Comment (kalachandrasekar1) [reply src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/manualdetermination/ClaimantLipManualDeterminationFormGenerator.java]: fixed

**Review by jarekPierchala — APPROVED** [review]

**Review by ElifDenizEsea — DISMISSED** [review]

**Review by portizdev — APPROVED** [review]

**Review by kenneth-hmcts — DISMISSED** [review]

**Review by neeta-hmcts — APPROVED** [review]

**Review by manish14836 — APPROVED** [review]

**Pull request merged**